package.json

```diff
@@ -7,9 +7,11 @@
     "dev:client": "NODE_ENV=development webpack -w",
     "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
     "prod:build": "NODE_ENV=production webpack -p",
-    "format": "prettier --write 'src/**/*.js'",
-    "lint": "eslint 'src/**/*.js'",
-    "test": "npm run lint",
+    "format": "prettier --write '{src,test}/**/*.js'",
+    "lint": "eslint '{src,test}/**/*.js'",
+    "pretest": "npm run lint",
+    "test": "mocha '{src,test}/**/*.test.js'",
+    "test:watch": "chokidar '**' --silent -c 'npm test'",
```

Comment:
Thanks for the tip! Initially filtered to src/ and test/ which means package, module, and other changes outside these directories will be undetected. File deletions do not appear to be observed.